server/controllers/user/get-supervised-tickets.php

```diff
@@ -103,12 +105,12 @@ public function createAuthorsArray(){
         
         if(!empty($this->supervisedUserList)){
             foreach(array_unique($this->supervisedUserList) as $supervised){ 
-                array_push($authors,['id'=> $supervised,'isStaff'=> 0]);
+                array_push($authors, ['id' => $supervised, 'isStaff' => 0]);
             }
         };
         
         if(!in_array( $user->id, $this->supervisedUserList) && $this->showOwnTickets){
-            array_push($authors,['id'=> $user->id*1,'isStaff'=> 0]);
+            array_push($authors, ['id' => $user->id*1, 'isStaff' => 0]);
```

Comment:
If you allow staff members, then the `'isStaff' => 0` part here must be corrected.

Reply:
@mredigonda what do you mean? Staff members can't be supervised. This information is here for the frontend standard of representing an user.

Reply:
That's correct, I was just looking at this syntactically but it doesn't make a lot of sense in real world. Anyway, now staffs are really not allowed in this path.